middlewares/accesslog/logger.go

```diff
@@ -31,10 +30,28 @@ type LogHandler struct {
 
 // NewLogHandler creates a new LogHandler
 func NewLogHandler(filePath string) (*LogHandler, error) {
-	if len(filePath) == 0 {
-		return nil, errors.New("Empty file name specified for accessLogsFile")
+	var file *os.File
+
+	if len(filePath) > 0 {
+		f, err := openAccessLogFile(filePath)
+		if err != nil {
+			return nil, err
```

Comment:
Good point, I added some specific error message that adds more context to the previous one.